resources/views/objects/_usercard.blade.php

```diff
@@ -94,7 +94,7 @@ class="user-action-button user-action-button--message"
             </div>
             <div class="usercard__status-bar usercard__status-bar--{{!isset($loading) && $user->isOnline() ? 'online' : 'offline'}}">
                 <span class="far fa-fw fa-circle usercard__status-icon"></span>
-                <span class="usercard__status-message" title="{{isset($loading) || $user->isOnline() ? '' : ($user->user_lastvisit ? trans('users.show.lastvisit', ['date' => $user->user_lastvisit->diffForHumans()]) : '')}}">
+                <span class="usercard__status-message" title="{{isset($loading) || $user->isOnline() || $user->hide_presence ? '' : trans('users.show.lastvisit', ['date' => $user->user_lastvisit->diffForHumans()])}}">
```

Comment:
probably should extract the check logic for the entire usercard block into a variable like `$showOnline = !isset($loading) && $user->isOnline();` and use that instead.